_dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue

```diff
@@ -46,26 +46,68 @@
             class="spinner"
           />
         </div>
-        <b-form-row
-          class="mt-2"
+        <b-container
           v-else
+          class=" maxw-sm-800"
         >
+          <div class="d-flex justify-content-around">
```

Comment:
It won't be responsive, is it an issue ?

Reply:
fixed as seen with product designer